.gitignore

```
@@ -1,3 +1,4 @@
+.idea
```

```suggestion
.idea
```

If you're interested in getting this checked in, would you mind sending a separate PR? I don't see any reason not to in general -- it's just good to keep PR changes as small as possible.

---

Cleaned it up 😏